### ISSUE TITLE
Win10 administrator startup can't drag and drop

### ISSUE THREAD

**Issue by chen2ding** [xuqiqiang/dragdropwindows#1]

**Comment by xuqiqiang** [comment]
In non-admin mode, files can be retrieved by drag and drop. But in admin mode, I can't get it, not even the WM_DROPFILES message.

Through searching, I found some reasons:

[Drag and Drop File into Application under run as administrator](http://superuser.com/a/59106)

Your drag'n'drop isn't working for permissions levels differences, if the other program/explorer you're dragging from is not elevated as administrator Windows is smart enough to not let the user do that.
[Can't drag and drop when 'run as administrator'](https://www.reddit.com/r/photoshop/comments/3j4j5b/cant_drag_and_drop_when_run_as_administrator/)

The problem is tied to how security permissions work. The ability to drag and drop from a normal applications to an elevated one would break the security model behind UAC. However, I'm unclear as to why a UAC prompt isn't shown, thus allowing for a temporary elevated operation (much like any linux user experiences every day). This is definitely something that Microsoft needs to work on. What troubles me is that this problem is already old. Vista behaved the same.

There's 2 things you can do (one ugly, one annoying)

    Ugly: Disable UAC. But you lose all the extra security it offers.
    Annoying: Use another File manager and run it too as an Administrator

Why another file manager? Because you can't actually elevate Windows Explorer. Despite seeing the option in the context menu of the Windows Explorer icon and a UAC prompt being displayed, the fact is Windows Explorer will not be elevated.

In a nutshell: Because of [UIPI](https://en.wikipedia.org/wiki/User_Interface_Privilege_Isolation), low-privileged processes cannot send messages to high-privileged processes.
